packages/dara-components/dara/components/smart/chat/chat.py

```diff
@@ -61,3 +63,14 @@ class Chat(StyledComponentInstance):
 
     class Config:
         extra = 'forbid'
+
+    @validator('value')
+    @classmethod
+    def validate_vaeiable(cls, value):
+        """
+        Validate that the Variable has a store attached to it.
+        """
+        if value.store is None:
+            raise ValueError('A Chat value variable must have a store attached to it')
```

Comment:
In the future we might add stores which are not `BackendStore`, e.g. for local storage or URL storage. This should check that the store is not None and an instance of BackendStore

Reply:
Added